client/common/common.c

```diff
@@ -181,6 +181,7 @@ usage(FILE *out, const char *name)
           " -C, --no-cursor       ignore cursor events.\n"
           " -T, --no-touch        ignore touch events.\n"
           " -K, --no-keyboard     ignore keyboard events.\n"
+          " --vim                 use vim bindings.\n"
```

Comment:
I think we should use --binding option with vim as argument instead. Some people may want to add their own bindings as well and there's already quite a lot of these options.

Reply:
Good call. I changed it in 6e1f83dc. I hope the implementation is satisfactory.